### DISCUSSION TITLE
Research papers on Eclipse GLSP

### DISCUSSION THREAD

**Discussion by KesThav** [eclipse-glsp/glsp#627] [General] [answered]
Hi everyone,

I am a master student in Business Informatics and i am currently writing a research papers on web-based metamodeling tools. I am looking for research papers on Eclipse GLSP but haven't found anything. I know that articles are available online (on the official page, EMF. could and more) but using online materials is prohibited by my university. Could you suggest me any research papers on Eclipse GLSP (if any exists) ?

Thank you,

Best regards,

**Comment by planger** [comment, marked as answer]
Hi,

I'm just aware of those two theses:
  - https://web.archive.org/web/20210324191359id_/https://repositum.tuwien.at/bitstream/20.500.12708/17040/1/Eder%20Hansjoerg%20-%202021%20-%20Towards%20debugging%20facilities%20for%20graphical%20modeling...pdf
  - https://model-engineering.info/publications/theses/thesis-forstner.pdf

I know that more papers are work in progress, but they aren't published yet.

Best wishes,

Philip

**Comment by KesThav** [comment]
Hi, thank you very much for your help

Best regards
